### PR TITLE
Migrate to pytest-selenium

### PR DESCRIPTION
**Pull request by davehunt**
Please review but do not merge - we will need to update all Jenkins jobs before we can merge.

### CODE REVIEW

**Comment by davehunt** [comment]
Adhoc runs:
* desktop: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/amo.adhoc.pytest-selenium/3
* mobile: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/amo.mobile.adhoc.pytest-selenium/3

File: pages/desktop/discovery.py [hunk 1]
```diff
@@ -35,23 +35,9 @@ class DiscoveryPane(Base):
 
     _featured_addons_base_locator = (By.CSS_SELECTOR, '#featured-addons .addon-title ')
 
-    def __init__(self, testsetup, path):
```
Comment (bobsilverberg) [inline pages/desktop/discovery.py]: So have you just done away with this whole `services_base_url` thing? I cannot claim to have ever understood what it was about, but I do see that it still exists in `conftest.py`.

**Comment by bobsilverberg** [comment]
This LGTM. Thanks for all that hard work, @davehunt. I haven't run the tests locally as I feel that the adhoc runs are trustworthy enough. r+

File: pages/desktop/discovery.py (hunk 1 above)
Comment (davehunt) [inline pages/desktop/discovery.py]: It's still there, but it uses a fixture now. Really, these tests just have a different base URL for production, and that's what this is for. I would suggest simply running a separate suite for just these tests and setting the different base URL, except there's one test that uses the application base URL and the services base URL. There's a good chance we can revisit to see if this is the right thing to do, but I wanted as few changes as possible in this particular patch.

**Comment by davehunt** [comment]
Thanks @bobsilverberg, I'll prepare Jenkins and merge this later today.

**Pull request merged**